.changeset/wild-readers-relate.md

```diff
@@ -0,0 +1,5 @@
+---
+"thirdweb": minor
+---
+
+Adds account linking to the Connect UI
```

Comment:
could condense these changesets

Reply:
Different features should be different changesets, right?